packages/python/package.json

```diff
@@ -25,7 +25,6 @@
     "@types/node": "14.18.33",
     "@vercel/build-utils": "6.7.1",
     "@vercel/ncc": "0.24.0",
-    "execa": "^1.0.0",
-    "typescript": "4.3.4"
+    "execa": "^1.0.0"
```

Comment:
**non-blocking:** We should consider updating to `execa@7.x` (currently 7.1.1). v1.x is 5 years old. Plus we should pin the version.